packages/@react-aria/overlays/test/useOverlay.test.js

```diff
@@ -11,57 +11,101 @@
  */
 
 import {fireEvent, render} from '@testing-library/react';
+import {installMouseEvent, installPointerEvent} from '@react-spectrum/test-utils';
+import {mergeProps} from '@react-aria/utils';
 import React, {useRef} from 'react';
 import {useOverlay} from '../';
 
 function Example(props) {
   let ref = useRef();
-  let {overlayProps} = useOverlay(props, ref);
-  return <div ref={ref} {...overlayProps} data-testid={props['data-testid'] || 'test'}>{props.children}</div>;
+  let {overlayProps, underlayProps} = useOverlay(props, ref);
+  return (
+    <div {...mergeProps(underlayProps, props.underlayProps || {})}>
```

Comment:
Why isn't `useOverlay` handling the merging of `props.underlayProps` into the returned `underlayProps`.

Reply:
there's no such thing as props.underlayProps, it's only for this test